build.py

```diff
@@ -641,6 +643,41 @@ def build(self):
 
 Project.add(Project_glib_openssl())
 
+class Project_graphene(GitRepo, Project):
+    def __init__(self):
+        Project.__init__(self,
+            'graphene',
+            repo_url = 'https://github.com/ebassi/graphene',
+            fetch_submodules = False,
+            tag = None,
+            dependencies = ['ninja', 'meson', 'pkg-config', 'glib'],
+            )
+
+    def build(self):
+        # where we build, with ninja, the library
+        ninja_build = os.path.join(os.path.join(self.build_dir, '_build'))
+        # First we check if we need to generate the meson build files
+        #   Note: actually, with the git checkout, we always rebuild everything because the _build dir is deleted also on the update
+        if not os.path.isfile(os.path.join(ninja_build, 'build.ninja')):
+            self.builder.make_dir(ninja_build)
+            # debug info
+            if self.builder.opts.configuration == 'release':
+                add_opts = '--buildtype release'

```

Comment:
I don't see that the options are equals, with meson you have more options (plain, debugoptimized, minsize)